frontend/graphql/posts.graphql

```diff
@@ -1,5 +1,5 @@
-query posts($authorId: Int!) {
-  posts(authorId: $authorId) {
+query posts($authorId: Int!, $status: PostStatus) {
```

Comment:
We should also make the `$status` arg required here in accordance with the backend request above